src/Component.ts

```
@@ -73,6 +73,42 @@ export class Component<T extends HTMLElement = HTMLElement> extends BaseComponen
+    /**
+     * Set style properties
+     * @param properties Object of style property name and value pairs
+     */
+    public css(properties: Record<string, string>): typeof this;
+
+    public css(...args: [string, string] | [string, string, boolean] | [Record<string, string>]): typeof this {
+        if (args.length === 2 || args.length === 3) {
+            const name: string = args[0],
+                value: string = args[1],
+                priority: boolean = args[2] ?? false;
+            this.element.style.setProperty(name, value, priority ? "important" : void 0);
```

Biome

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)